### DISCUSSION TITLE
Chart tdk muncul di Shiny Dashboard tapi tdk ada error message

### DISCUSSION THREAD

**Discussion by Nava2022** [teamalgoritma/community#90] [Q&A] [answered]
Halo,
Mau tanya untuk tugas LBB IP. 
Saya sudah create project baru dg Shiny web app dan bikin file ui, server, global terpisah. Sebelumnya semua chart ggplotly sudah di run di RMarkdown dan berhasil semua, lalu saya copy-paste ke server. Tapi ketika di run app, chart nya tidak muncul di Shiny dashboard. Tolong dibantu. Terimakasih,
Nava
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547146-1b16b5aa-28a3-4dd5-a8d3-80281d48f3f6.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547185-97c346c6-8ce4-4a35-8a2e-1bbab8317784.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547227-7b8f0198-87b9-4d95-849b-81dad903c36d.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547271-f7290815-9117-4923-8168-c4bb651a369e.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547349-3f000355-47aa-4d1a-a184-ffeb0a60f888.png">

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167547407-3661bbf6-86d0-4cac-b090-eaf9414ce6fe.png">

**Comment by inytss** [comment]
Dear @Nava2022 ,
Apabila saya lihat berdasarkan code yang Ibu buat sepertinya nama tabname untuk masing-masing menu menggunakan spasi. Ibu dapat menggunakan underscore (_) sebagai pengganti spasi. Pastikan pemberian nama object baik object plot, tabname, ataupun nama inputId dan outputId jangan ada yang menggunakan spasi ataupun memiliki nama yang sama.
Semoga membantu :)

**Comment by Nava2022** [comment]
Terimakasih banyak. Sudah berhasil. Tapi ada masalah lagi. Barusan deploy completed, tapi di shinyapp masih error. Kenapa ya?

![image](https://user-images.githubusercontent.com/103920599/167622213-7abe3ce6-113f-479a-aade-fc9ef99e43af.png)

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/103920599/167622278-21befe1f-21be-4bff-9790-27979b65ede5.png">

**Comment by VictorNugraha** [comment, marked as answer]
Halo, @Nava2022!

Untuk melihat error yang terjadi, Ibu bisa menunjungi bagian **_logs_** pada wesbite shinyapps.io. Berikut cara menuju bagian _**logs**_,

1. Bukalah bagian Dashboard, lalu pilihah app Shiny yang telah Ibu upload. Sebagai contoh, nama applikasi saya InvestNow.
<img width="885" alt="Capture" src="https://user-images.githubusercontent.com/28379954/167803870-61fc2c2b-7be5-41fc-b07f-e2b0ad3f3ef4.PNG">

2. Setelah masuk, bisa memilh bagian logs 

<img width="768" alt="Capture" src="https://user-images.githubusercontent.com/28379954/167805525-8ae34a9a-5ad1-48f4-a4a8-4415448e3e8a.PNG">

3. Nantinya dibagian logs bisa melihat informasi error yang terjadi.

<img width="756" alt="Capture2" src="https://user-images.githubusercontent.com/28379954/167805732-f849075e-c3b8-4bab-a584-2b7b68c7a514.PNG">

Demikian infromasi yang dapat saya sampaikan. Semoga membantu.
